test/test.js

```diff
@@ -3299,7 +3300,9 @@ function tests(dbName, dbType, viewType) {
         });
 
         // make sure prefixed DBs are tied to regular DBs
-        var db = new Pouch(dbName, {prefix: 'myprefix_'});
+        var dbBaseName = path.basename(dbName);
+        var prefix = path.dirname(dbName) + '/myprefix_';
+        var db = new Pouch(dbBaseName, {prefix: prefix});
```

Comment:
^ This is actually kinda dangerous, because it occasionally causes bugs related to the `prefix` itself. But I agree that cleaning up the dirty git workspace is annoying, and we ought to be testing `prefix`, so I'm +1 on this.